File: Ruby/DataSetSlection/UnionOfSets/step1.rb
# N 個の要素からなる数列Aと数値Bが与えられます。BがAに含まれているか判定してください。

n, b = gets.split(" ").map(&:to_i)
a = gets.chomp.split.map(&:to_i)

if a.include?(b)
    puts "Yes"
else
    puts "No"
end

File: Python/ConditionalBranch/And+Or/step1.py
# 2 つの整数A、Bが与えられます。AとBが両方とも 10 以上の場合はYESを、そうではない場合はNOを出力してください。

a,b = map(int, input().split())

if a >= 10 and b >= 10:
    print("YES")
else:
    print("NO")